tutorials/light_ecModel/protocol.m

```diff
@@ -0,0 +1,143 @@
+% full ecModels. However, no custom kcat values are defined for the ecModel
+% in this tutorial.
+
+% STEP 11
+% getKcatAcrossIsoenzymes cannot be run on light ecModels
+
+% STEP 12
+% getStandardKcat can be run on light ecModels in a similar way as for full
+% ecModels. But due to the different model structure, no "standard"
+% pseudo-enzyme is included 
+% 
+% Instead of introducing a "standard" pseudo-enzyme, the new
+% standard enzyme usage constraints follow the light ecModel structure,
+% where the prot_pool is included as pseudo-substrate with
+% -standardMW/standardKcat as the stoiciometric coefficient.
+[ecModel, rxnsMissingGPR, standardMW, standardKcat] = getStandardKcat(ecModel);
+
+% STEP 13
+% In light ecModels the kcat values are also kept in ecModels.ec.kcat, and
+% only after running applyKcatConstraints are these introduced in the
+% ecModel.S matrix. In constrast to full ecModels, the applyKcatConstraints
+% function checks across isoenzymes which is the most efficient enzyme
+% (lowest MW/kcat value), and uses this value.
+ecModel = applyKcatConstraints(ecModel);
+
+% STEP 14
+ecModel = setProtPoolSize(ecModel);
+sol=solveLP(ecModel,1)
+printFluxes(ecModel,sol.x)
+saveEcModel(ecModel);
+
+% STEP 16
+% sensitivityTuning also works on light ecModels. However, in this tutorial
+% an ecModel for the generic Human-GEM model is generated. This model is
+% not directly suitable for simulations, as it does not represent a
+% specific cell-type. It is therefore not trivial to set what exchange
+% reactions should be open (to represent the culture medium [for cultured
+% cells, or otherwise the natural environment]), and what maximum growth
+% rate should be reached.
+
+% STEP 18-21
+% Proteomics integration is NOT possible with light ecModels.
+
+% STEP 23
+% Simulating fluxes in light ecModels would follow a similar strategy as
+% would be suitable for simulating full ecModels without proteomics
+% integration. E.g. first optimization of a "classical" objective function
+% (e.g. maximize growth rate, maximize ATP turnover, or minimize glucose
+% uptake), constraining the ecModel with the objective value that was
+% reached, followed by a second optimization where the total enzyme usage
+% is minimized, yielding the most efficient enzyme allocation.
+ecModel = setParam(ecModel,'obj',adapter.params.bioRxn,1);
+sol = solveLP(ecModel)
+disp(['Growth rate reached: ' num2str(abs(sol.f))])
+% Set growth lower bound to 99% of the previous value
+ecModel = setParam(ecModel,'lb',adapter.params.bioRxn,0.99*abs(sol.f));
+% Minimize protein pool usage. As protein pool exchange is defined in the
+% reverse direction (with negative flux), minimization of protein pool
+% usage is computationally represented by maximizing the prot_pool_exchange
+% reaction.
+ecModel = setParam(ecModel,'obj','prot_pool_exchange',1);
+sol = solveLP(ecModel)
+disp(['Minimum protein pool usage: ' num2str(abs(sol.f)) ' mg/gDCW'])
+
+
+% TODO: LAST FEW STEPS
+
```

Comment:
I believe it has become urgent to merge this PR. For this outstanding addition here, I propose to create a separate issue, thus resulting in a small PR that will be easier to diff and review.

```suggestion
```